tests/theseus_tests/test_misc.py

```diff
@@ -0,0 +1,15 @@
+import pytest
```

Comment:
license header missing

Maybe we can add the flake8-copyright thing to Theseus?

Reply:
Oh yeah, I forgot about that, I'll add it.